app/views/nic/_base_form.html.erb

```diff
@@ -4,7 +4,16 @@
 
 <%= text_f f, :mac, :class => :interface_mac %>
 <%= text_f f, :identifier, :disabled => !f.object.new_record?,
-              :help_inline => _("Device identifier, i.e.: eth0 or eth1.1"),
+              :help_inline => popover(
+                      _("Device identifier"),
+                      _("Device identifier for this interface, this may be different on various platforms and environments, here are some common examples<br/><ul>" +
+                                "<li>For physical interfaces you can use <strong>eth0</strong> or if you're using biosdevname then <strong>em0.</strong></li>" +
+                                "<li>For virtual interfaces you must either use Alias notation <strong>eth0:0</strong> or VLAN notation <strong>eth0.1</strong>, " +
```

Comment:
`For virtual interfaces, use either alias notation (<strong>eth0:1</strong>, name:index) or VLAN notation (<strong>eth0.15</strong>, name:tag).` and remove the next line